.github/workflows/check-docker-image-digests.yml

```diff
@@ -0,0 +1,17 @@
+name: Check Docker Image Digests
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 5 * * 1-5'
```

Comment:
What is the reasoning for this? Should we not check this every day?

Reply:
basically this was copied from the integration tests pipeline to only run on working days, but of course with such a short running pipeline it should also be fine to just run it every night